README.md

```diff
@@ -1,132 +1,184 @@
----
-tags: OOP, properties, methods
-language: objc
----
+# Object Oriented People
 
-## Learning Goals
-  1. Learn how Objects work
-  2. Get better with Objective-C
+## Objectives
 
-## Requirements
-  1. Create a new Person Class.  Person should be an NSObject subclass.  
-  2. Add all of the required properties to the Person class 
-  3. Implement all of the required methods in the Person class 
+1. Create a new class that contains read-only properties.
+2. Write a designated initializer, default initializer, and convenience initializer to set initial values for the properties of an instance of that class.
+3. Write public methods that provide a controlled interface with the read-only properties.
+ 
+## Instructions
 
-### Person
+Fork and clone this lab. 
 
-#### About Properties
-Properties provide a way to create instance variables and accessor methods (getters and setters) in a single step.  Say for instance we created a class called cats.  If Cats has a property (of type NSNumber) numberOfLegs, the cat class will automatically, behind the scenes generate two methods for us: 
+Open the `object-oriented-people.xcworkspace` file. 
 
-```objc 
+### I. Create the Class Files
 
- - (NSNumber *)numberOfLegs; // getter
- - (void)setNumberOfLegs:(NSNumber *)numberOfLegs; // setter
+Create a new class called `FISPerson` that inherits from `NSObject`.
 
-```
+#### Declare the Properties
+
+Publicly declare four properties as `readonly` properties:
+
+  * An `NSString` property called `name`,
+  * An `NSUInteger` property called `age`,
+  * An `NSMutableArray` property called `skills`, and
+  * A `BOOL` property called `qualifiedTutor`.
 
-We can use these methods to both set and get our class's properties.  Let's say we create a Cat 'myCat' in our AppDelegates' didFinishLaunchingWithOptions method.  We can set the cats number of legs and get the cats number of legs using the following code.  
+**Top-tip:** *Remember that primitive-type properties cannot accept a retention attribute (*`strong`/`retain` *or* `weak`*).*
```

Comment:
`retain` is a relic at this point... maybe say "(`strong`/`weak`, or the antiquated `retain`)"